src/sinks/aws_cloudwatch_logs/mod.rs

```diff
@@ -146,29 +150,31 @@ pub enum CloudwatchError {
 impl SinkConfig for CloudwatchLogsSinkConfig {
     fn build(&self, cx: SinkContext) -> crate::Result<(super::RouterSink, super::Healthcheck)> {
         let batch = self.batch.unwrap_or(1000, 1);
-        let request = self.request.unwrap_with(&REQUEST_DEFAULTS);
+        // let request = self.request.unwrap_with(&REQUEST_DEFAULTS);
 
         let log_group = self.group_name.clone();
         let log_stream = self.stream_name.clone();
 
         let svc = ServiceBuilder::new()
-            .concurrency_limit(request.in_flight_limit)
+            // .concurrency_limit(request.in_flight_limit)
+            .concurrency_limit(1)
```

Comment:
This is what I do not understand. If I return commented line back, then test `cloudwatch_insert_log_event_batched` will hang up tests.
I think this related with `CloudwatchLogsSvc::poll_ready` because context in oneshot receiver rewritten by another `poll_ready` call.